src/doc/rustc-dev-guide/src/building/bootstrapping/debugging-bootstrap.md

```diff
@@ -0,0 +1,99 @@
+# Debugging bootstrap
+
+> FIXME: this page could be expanded 
+
+## `tracing` in bootstrap
+
+bootstrap has conditional [`tracing`][tracing] setup to provide structured logging.
+
+[tracing]: https://docs.rs/tracing/0.1.41/tracing/index.html
+
+### Enabling `tracing` output
+
+bootstrap will conditionally enable `tracing` output if the `BOOTSTRAP_TRACING` env var is set.
```

Comment:
```suggestion
Bootstrap will conditionally enable `tracing` output if the `BOOTSTRAP_TRACING` env var is set.
```

Reply:
Oops forgor to reply, fixed the capitlizations and also fixed a sentence that abruptly ended.